doc/examples/segmentation/plot_regionprops.py

```diff
@@ -3,15 +3,23 @@
 Measure region properties
 =========================
 
-This example shows how to measure properties of labelled image regions.
+This example shows how to measure properties of labelled image regions. It is
+a toy example, where the image regions are two ellipses. The outputs are the
```

Comment:
An example is an example :). I would modify like this:
`It is a toy example, ...` -> `We analyze an image with two ellipses.`

Reply:
Agreed :)
Included in fd4bcfa -- I manually created an "Apply suggestions from code review" commit with you @soupault as a co-author, hope that's all right!